python/lib/eeg.py

```diff
@@ -355,9 +357,9 @@ def fetch_and_insert_eeg_file(self, derivatives=None):
         # grep the raw files from eeg_files list
         eeg_file = BidsReader.grep_file(
             files_list         = files_list,
-            match_pattern      = ".(set$|edf$|vhdr$|cnt$|vsm$|bdf$)",
+            match_pattern      = ".(set$|edf$|vhdr$|vmrk$|eeg$|bdf$)",
```

Comment:
this is the final approved list of accepted file extensions for EEG datasets in BIDS